pyomyo.py

```diff
@@ -43,6 +43,8 @@
 import sys
```

Comment:
Be sure not to confuse the ideas of concurrencyâstructuring a program as independently executing componentsâand parallelismâexecuting calculations in parallel for efficiency on multiple CPUs. Although the concurrency features of Go can make some problems easy to structure as parallel computations, Go is a concurrent language, not a parallel one, and not all parallelization problems fit Go's model. For a discussion of the distinction, see the talk cited in this blog post.

📄 [Concurrency Parallelization](https://app.semasoftware.com/guides/613b974794340b2a26f11288/concurrency-parallelization-750)

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix